Can't resolve 'rxjs/observable/fromEvent' still occurs without rxjs-compat

Related to #20 and #21 

It seems like there is still an issue. Using rxjs 6.5.2. This library won't work without also adding rxjs-compat. It seems like the other tickets suggest that rxjs-compat is not required and that updating rxjs fixes the issue. However I find that not to be the case.

```
ERROR in ./node_modules/ng2-animate-on-scroll/dist/src/scroll.service.js
Module not found: Error: Can't resolve 'rxjs/observable/fromEvent' in 'node_modules/ng2-animate-on-scroll/dist/src'
```

The issue I believe is [here](https://github.com/abhazelton/animate-on-scroll/blob/master/src/scroll.service.ts#L3) 

`import { fromEvent } from 'rxjs/observable/fromEvent';`

should be 

`import { fromEvent } from 'rxjs';`

--- a/src/scroll.service.ts
+++ b/src/scroll.service.ts
@@ -1,6 +1,5 @@
 import { Injectable, OnDestroy } from '@angular/core';
-import { Observable, Subscription } from 'rxjs';
-import { fromEvent } from 'rxjs/observable/fromEvent';
+import { Observable, Subscription, fromEvent } from 'rxjs';
 
 @Injectable()
 export class ScrollService implements OnDestroy {

